plugins/CoreHome/CoreHome.php

```diff
@@ -413,6 +413,7 @@ public function getClientSideTranslationKeys(&$translationKeys)
         $translationKeys[] = 'CoreHome_CopyModalNote';
         $translationKeys[] = 'CoreHome_CopyX';
         $translationKeys[] = 'CoreHome_CopyXDescription';
+        $translationKeys[] = 'General_YourSessionHasExpired';
```

Comment:
Loading a translation into javascript just for using it in UI tests might not be that valuable.
```suggestion
```